composer.json

```diff
@@ -46,7 +46,7 @@
         "ext-openssl": "*",
         "ext-session": "*",
         "ezyang/htmlpurifier": "^4.17",
-        "henrique-borba/php-sieve-manager": "^1.0",
+        "henrique-borba/php-sieve-manager": "~1.0.4",
```

Comment:
We don't want such a hard constraint to this lib. It will prevent upgrades of the lib for non-major versions as 1.1.0.